Resources/views/CRUD/action_buttons.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% spaceless %}
-    {% for item in admin.getActionButtons(action, object|default(null)) %}
+    {% for item in admin.getActionButtons(action, (object is defined and object is not null)?object:null ) %}
```

Comment:
* Add a Twig comment to quickly explain why we should not use `default`
* Add spaces on your ternary condition: https://twig.sensiolabs.org/doc/2.x/coding_standards.html